common/xbps-src/shutils/build_dependencies.sh

```diff
@@ -368,6 +383,11 @@ install_pkg_deps() {
                 echo "   [runtime] ${_realpkg}: not found."
             fi
         fi
+        if [ "${_realpkg}" = "$targetpkg" ]; then
```

Comment:
Won't that break packages like wesnoth that depend on their -data subpackage ?

Reply:
Let me test

Reply:
No because self subpkgs are ignored.

Reply:
Verified, if you've got any other case just let me know.

Tested from scratch xbps build.
Tested wesnoth.
Tested extra-cmake-modules/kcoreaddons with the build loops

All success